happybase/connection.py

```diff
@@ -9,11 +9,14 @@
 from thrift.transport.TSocket import TSocket
 from thrift.transport.TTransport import TBufferedTransport, TFramedTransport
 from thrift.protocol import TBinaryProtocol, TCompactProtocol
+import sasl
```

Comment:
Dependencies are the cyrus-sasl extension. Those are available on every reasonable recent *nix system